crates/re_viewport/src/space_view_heuristics.rs

```diff
@@ -27,6 +27,10 @@ fn is_spatial_class(class: &SpaceViewClassName) -> bool {
     class.as_str() == "3D" || class.as_str() == "2D"
 }
 
+fn is_spatial_2d_class(class: &SpaceViewClassName) -> bool {
+    class.as_str() == "2D"
```

Comment:
```suggestion
    class.as_str() == SpatialSpaceView2D::NAME
```

The function above should follow the same pattern.

Reply:
Relevant: https://github.com/rerun-io/rerun/pull/4386

Reply:
After discussion decided not to make this change since it requires picking up an unwanted dependency -- will refactor away in https://github.com/rerun-io/rerun/issues/4388